apps/src/gamelab/AnimationTab/AnimationTab.jsx

```diff
@@ -62,13 +65,13 @@ var AnimationTab = React.createClass({
             <AnimationList />
           </div>
           <div style={styles.framesColumn}>
-            <div className="purple-header workspace-header">
+            <div className="purple-header workspace-header" style={commonStyles.purpleHeader}>
               <span>Frames</span>
             </div>
             <FrameList />
           </div>
           <div style={styles.editorColumn}>
-            <div className="purple-header workspace-header">
+            <div className="purple-header workspace-header" style={commonStyles.purpleHeader}>
```

Comment:
I now have a lot of the workspace-header styles defined inline for CodeWorkspace. I can probably use those here (and the couple other places they are used) too, but wanted to do the minimally impacting change to start. I continue to use the purple-header class since it still defines some descendant styles.